EnglishFilter/sections/specific.txt

```diff
@@ -2097,7 +2097,7 @@ masterani.me##a[href^="https://www.masterani.me/out/"] > img
 ||masterani.me/static/neenee/*-*.$image
 sunderlandecho.com##.slab__block > #adSlotMainContent1 + .backfill-cta
 history.com##.m-header-ad
-history.com##.m-in-content-ad-row
+history.com##aside.m-in-content-ad-row
```

Comment:
is it not applies without tag?

Reply:
yes, it didn't work on my end without tag or if rule was be more specific like `history.com##.m-detail--body > .m-in-content-ad-row`.